src/Mimi.jl

```diff
@@ -37,6 +37,7 @@ export
     getproperty,
     get_parameter_value,
     get_variable_value,
+    hasvalue,
```

Comment:
I wonder whether we should export this?

Reply:
It isn't clear to me whether `hasvalue` is ever needed in `run_timestep` functions. It's currently used only in the connector components.